### PR TITLE
dev-java/jansi: bump to 2.4.0

### PR DESCRIPTION
**Pull request by vaukai**

### CODE REVIEW

File: dev-java/jansi/jansi-2.4.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+JAVA_MAIN_CLASS="org.fusesource.jansi.AnsiMain"
+
+# junit-jupiter is not available in ::gentoo
+#JAVA_TEST_GENTOO_CLASSPATH="!!!artifactId-not-found!!!,!!!groupId-not-found!!!,!!!groupId-not-found!!!"
+#JAVA_TEST_SRC_DIR="src/test/java"
+#JAVA_TEST_RESOURCE_DIRS="src/test/resources"
+
+src_prepare() {
+	default
+	rm -r "${JAVA_RESOURCE_DIRS}/org/fusesource/jansi/internal/native" || die
```
Comment (fordfrog) [inline dev-java/jansi/jansi-2.4.0.ebuild:54]: could you please comment why we need to remove this?

**Comment by fordfrog** [comment]
merged everything except `dev-java/jansi` as we need to clarify the code removal

File: dev-java/jansi/jansi-2.4.0.ebuild (hunk 1 above)
Comment (vaukai) [reply dev-java/jansi/jansi-2.4.0.ebuild:55]: Done https://github.com/gentoo/gentoo/compare/d38e0fde4bd3157d380e225b33ea60b745ff8688..078d9cc32c7bbb94c0349066b773ccb511fb58a8#diff-1ce1c8af1dc228f0228e8b8a5b31bc719c5cef45e6cd5658633f8c6119f350b3

**Comment by fordfrog** [comment]
@vaukai merged, thanks!

**Pull request closed without merging**